common/src/Polyhedron.h

```diff
@@ -392,12 +414,12 @@ class Polyhedron {
     bool checkEdges() const;
     bool checkEdgeLengths(const T minLength = vm::constants<T>::pointStatusEpsilon()) const;
```

Comment:
This one is inconsistent with `healEdges` using 0.01

Reply:
done

Reply:
good find, too